File: LetsPlant/app/src/main/java/com/e/letsplant/data/Post.java
package com.e.letsplant.data;

import com.e.letsplant.PostModel;

import java.util.ArrayList;

public class Post {
    private String ownerName;

    public Post(String ownerName) {
        this.ownerName = ownerName;
    }

    public String getOwnerName() {
        return this.ownerName;
    }
}

File: LetsPlant/app/src/main/java/com/e/letsplant/data/Post.kt
package com.e.letsplant.data

class Post {
    var postId: String? = null
    var postImage: String? = null
    var description: String? = null
    var publisher: String? = null

    constructor()
    constructor(postId: String?, postImage: String?, description: String?, publisher: String?) {
        this.postId = postId
        this.postImage = postImage
        this.description = description
        this.publisher = publisher
    }
}